ui/lib/core/addon/components/page/error.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="box is-sideless has-background-white-bis has-text-grey has-text-centered has-tall-padding" data-test-page-error>
+  {{#if (eq @error.httpStatus 404)}}
+    <h1 class="title is-3 has-text-grey">
+      404 Not Found
+    </h1>
+    <p>Sorry, we were unable to find any content at <code>{{@error.path}}</code>.</p>
+  {{else if (eq @error.httpStatus 403)}}
+    <h1 class="title is-3 has-text-grey">
+      Not authorized
+    </h1>
+    <p>You are not authorized to access content at <code>{{@error.path}}</code>.</p>
+  {{else}}
+    <h1 class="title is-3 has-text-grey">
+      Error
+    </h1>
+    <p>
+      {{#if @error.message}}
+        <p data-test-page-error-message>{{@error.message}}</p>
+      {{/if}}
+      {{#each @error.errors as |error index|}}
+        <p data-test-page-error-details={{index}}>{{error}}</p>
+      {{/each}}
+    </p>
+  {{/if}}
+</div>
```

Comment:
I took this from PKI and moved it to the core addon for reuse.

Reply:
🎉